WooCommerce/Woo Watch App/Orders/OrderDetailLoader.swift

```diff
@@ -66,7 +66,7 @@ final class OrderDetailLoaderViewModel: ObservableObject {
         self.viewState = .loading
         let dataService = OrderNotificationDataService(credentials: dependencies.credentials)
         do {
-            let (_, remoteOrder) = try await dataService.loadOrderFrom(noteID: pushNotification.noteID)
+            let (_, remoteOrder) = try await dataService.loadOrderFrom(notification: pushNotification)
```

Comment:
Passing the entire `pushNotification` so we can do the loading of the order from the `note_full_data` part of the payload